src/Modules/OAuth/Domain/Settings/Command/UpdateModuleSettingsHandler.php

```diff
@@ -0,0 +1,42 @@
+<?php
+
+namespace ForkCMS\Modules\OAuth\Domain\Settings\Command;
+
+use ForkCMS\Core\Domain\MessageHandler\CommandHandlerInterface;
+use ForkCMS\Modules\Extensions\Domain\Module\ModuleName;
+use ForkCMS\Modules\Extensions\Domain\Module\ModuleSettings;
+use Symfony\Component\Messenger\Attribute\AsMessageHandler;
+
+class UpdateModuleSettingsHandler implements CommandHandlerInterface
+{
+    public function __construct(private readonly ModuleSettings $moduleSettings)
+    {
+    }
+
+    public function __invoke(UpdateModuleSettings $updateSettings): void
+    {
+        $this->moduleSettings->set(
+            ModuleName::fromString('OAuth'),
+            'client_id',
```

Comment:
Zou je deze niet prefixen met azure? Kwestie als er ooit andere zijn dat dit geen issue gaat geven? Of is dat volgens jou generiek genoeg om herbruikbaar te zijn voor verschillende oAuth implementaties?

Reply:
fixed d97a5666e898c8345dda92d9f02ea746e9a060ae